dev-client/src/screens/SoilScreen/components/EditDepthModal.tsx

```diff
@@ -176,7 +176,7 @@ export const EditDepthModal = ({
           onPress={onOpen}
         />
       )}
-      Header={
+      heading={
         <Heading variant="h6">
           {mutable ? t('soil.depth.edit_title') : renderDepth(t, thisDepth)}
```

Comment:
I was tempted to switch this to a `TranslatedHeader`, but since the immutable case directly renders a string (which that component isn't yet designed to handle) I thought it'd be better to defer this until we tackle our typography design tickets.